src/redux/Weather/ActionCreators.js

```diff
@@ -0,0 +1,39 @@
+import { FETCH_WEATHER, DISPLAY_COUNTRIES } from './Actions';
+import WEATHER_API from '../../api.config';
```

Comment:
- [ ] You should get data from a public API and use a Filter stateless component to filter data retrieved from the API, please consider implementing this functionality so your app is consuming the API properly